DEVELOPMENT.md

```diff
@@ -0,0 +1,8 @@
+## Documentation on building the SDK
+To do development on the SDK source, use the following commands:
+
+```
+brew install cocoapods # if you do not have cocoapods installed
+pod install
```

Comment:
I get the following in the project root folder on this line:
```
❯ pod install
[!] No `Podfile' found in the project directory.
```

Reply:
actually this step was unneeded for me. Only step I had to do was the `xcpretty` was not installed so I had to run `sudo gem install xcpretty`. Was fairly straightforward to debug this so don't know if we need to explicitly list out the dependencies